### PR TITLE
nullable

### PR DESCRIPTION
**Pull request by serprex**
Allow nullable support to be enabled with env dynconf

Followup will want to allow per-mirror dynconf overrides. That'd make e2e testing easier

### CODE REVIEW

File: flow/peerdbenv/dynamicconf.go [hunk 1]
```diff
@@ -76,6 +76,12 @@ END;`, ValueType: protos.DynconfValueType_STRING,
 		ApplyMode:        protos.DynconfApplyMode_APPLY_MODE_AFTER_RESUME,
 		TargetForSetting: protos.DynconfTarget_ALL,
 	},
+	{
+		Name: "PEERDB_NULLABLE", DefaultValue: "false", ValueType: protos.DynconfValueType_BOOL,
```
Comment (Amogh-Bharadwaj) [inline flow/peerdbenv/dynamicconf.go:80]: I thought we want it be true by default? 
cc @iskakaushik @saisrirampur

Comment (heavycrystal) [reply flow/peerdbenv/dynamicconf.go:80]: shouldn't be, perf impact
needs to be opt-in

File: flow/connectors/bigquery/qvalue_convert.go [hunk 2]
```diff
@@ -12,7 +12,8 @@ import (
 
 func qValueKindToBigQueryType(columnDescription *protos.FieldDescription) bigquery.FieldSchema {
 	bqField := bigquery.FieldSchema{
-		Name: columnDescription.Name,
+		Name:     columnDescription.Name,
+		Required: columnDescription.NullableEnabled && columnDescription.Nullable,
```
Comment (heavycrystal) [inline flow/connectors/bigquery/qvalue_convert.go:16]: should be `!columnDescription.Nullable` I think

File: flow/peerdbenv/dynamicconf.go (hunk 1 above)
Comment (serprex) [reply flow/peerdbenv/dynamicconf.go:80]: also prefer to have new options not change existing behavior

File: flow/connectors/clickhouse/normalize.go [hunk 3]
```diff
@@ -83,27 +83,30 @@ func generateCreateTableSQLForNormalizedTable(
...
+		} else {
 			stmtBuilder.WriteString(fmt.Sprintf("`%s` %s, ", colName, clickhouseType))
 		}
 	}
 	// TODO support soft delete
 	// synced at column will be added to all normalized tables
 	if syncedAtColName != "" {
 		colName := strings.ToLower(syncedAtColName)
-		stmtBuilder.WriteString(fmt.Sprintf("`%s` %s, ", colName, "DateTime64(9) DEFAULT now()"))
+		stmtBuilder.WriteString(fmt.Sprintf("`%s` %s, ", colName, "DateTime64(9) DEFAULT now64()"))
```
[the remark this answers is not in the document]
Comment (serprex) [reply flow/connectors/clickhouse/normalize.go:103]: better to fix new raw tables in meantime so problem doesn't continue to grow

**Review by iskakaushik — APPROVED** [review]

**Pull request merged**